test/unit/property.test.js

```diff
@@ -215,9 +215,67 @@ describe('Property', function () {
     });
 
     describe('.replaceSubstitutions', function () {
+        /**
+         * Generates a object with n variables
+         *
+         * @example
+         * getVariables(2)
+         * {
+         *    '0': '',
+         *    '1': ''
+         * }
+         *
+         * @param {Integer} n - Number of variables
+         * @returns {Object}
+         */
+        function getVariables (n) {
+            var i,
+                obj = {};
+
+            for (i = 0; i < n; i++) {
+                obj[String(i)] = '';
+            }
+
+            return obj;
+        }
+
         it('should bail out if a non-string argument is passed', function () {
             expect(Property.replaceSubstitutions(['random'])).to.eql(['random']);
         });
+
+        it('should resolve poly chained variable 19 times', function () {
+            // eslint-disable-next-line max-len
+            var str = '{{20{{19{{18{{17{{16{{15{{14{{13{{12{{11{{10{{9{{8{{7{{6{{5{{4{{3{{2{{1{{0}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}}',
```

Comment:
This will have to be updated if the limit ever changes.

Reply:
Yes, any alternative?

Reply:
Have a function that accepts a number argument and builds this string for you.

>

Reply:
done